images/installer/root/usr/local/bin/user_setup

```diff
@@ -6,9 +6,15 @@ mkdir -p ${HOME}
 chown ${USER_UID}:0 ${HOME}
 chmod ug+rwx ${HOME}
 
+chown ${USER_UID}:0 ${HOME}/.ssh/config
+
 # runtime user will need to be able to self-insert in /etc/passwd
 chmod g+rw /etc/passwd
 
+# make required config dirs writable
+chown ${USER_UID}:0 ${HOME}/.kube/config
+chmod a+rw ${HOME}/.kube/config
```

Comment:
I suspect this file is bind-mounted, and thus this would have an impact on permissions outside of the container...